### PR TITLE
azurelinux-sysinfo: add libselinux-utils as a requirement

### PR DESCRIPTION
**Pull request by amritakohli**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] If you are adding/removing a .spec file that has multiple-versions supported, please add [@microsoft/cbl-mariner-multi-package-reviewers](https://github.com/orgs/microsoft/teams/cbl-mariner-multi-package-reviewers) team as reviewer [(Eg. golang has 2 versions 1.18, 1.21+)](https://github.com/microsoft/azurelinux/tree/2.0/SPECS/golang)
- [x] Ready to merge

---

###### Summary 

This PR adds libselinux-utils as a requirement in azurelinux-sysinfo package. It's needed for the getenforce command to work. 

###### Change Log  

- Add requirement libselinux-utils in azurelinux-sysinfo spec.

###### Does this affect the toolchain?  

**NO**

###### Test Methodology

- Tested in local VM with Mariner image
- Pipeline build id: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=592643&view=results

### CODE REVIEW

File: SPECS/azurelinux-sysinfo/azurelinux-sysinfo.spec [hunk 1]
```diff
@@ -13,6 +13,7 @@ Source2:        azurelinux-sysinfo.service
 Source3:        sysinfo-selinuxpolicies.cil
```
Comment (cwize1) [inline SPECS/azurelinux-sysinfo/azurelinux-sysinfo.spec:13]: Bump `Release` number?

Comment (reubeno) [reply SPECS/azurelinux-sysinfo/azurelinux-sysinfo.spec:13]: Good catch!

On this topic... @dmcilvaney @PawelWMS have we ever had a github action that checks for an inadvertently forgotten release bump? (i.e., Should be able to rpmspec-parse the .spec in the _from_ branch and the _to_ branch and at least require a difference.) Missing a bump can be problematic, and we should definitely make machines sort this out for us. 

(And before it's mentioned, I assume there's a problem of cases where someone is just updating changelog entries or making changes to comments, say, and that shouldn't strictly require a bump. That sounds like a case where someone could use an escape hatch of adding some well-known tag/line in the PR description that quiets the check. e.g., `IKnowWhatIAmDoingAndReallySwearUpAndDownThatThereAreNoMaterialChangesInThisSpecFile`. Or maybe something shorter if someone clever can come up with something 😄  )

Comment (amritakohli) [reply SPECS/azurelinux-sysinfo/azurelinux-sysinfo.spec:13]: Thank you for catching this!

**Review by pebenito — APPROVED** [review]

**Review by romoh — APPROVED** [review]
LGTM
Approved as a package install bug fix

**Pull request merged**